gestalt-inject-java/src/main/java/org/terasology/gestalt/annotation/processing/BeanDefinitionProcessor.java

```diff
@@ -86,11 +99,19 @@ public synchronized void init(ProcessingEnvironment processingEnvironment) {
         this.utility = new ElementUtility(processingEnvironment.getElementUtils(), processingEnvironment.getTypeUtils());
     }
 
+    /**
+     * {@return the supported annotation types.}
```

Comment:
```suggestion
     * {@inheritDoc}
```
This is an override from an already well-documented base method, so we might as well inherit that.